src/context/campaignConfigsStore.test.tsx

```diff
@@ -46,7 +46,7 @@ describe("CampaignConfigsStoreContextProvider", () => {
           {({ hasLoadedFromStore, allCampaignConfigs }) => (
             <>
               {hasLoadedFromStore && (
-                <Text testID="loaded">{hasLoadedFromStore}</Text>
+                <Text testID="loaded">{`${hasLoadedFromStore}`}</Text>
```

Comment:
Why do we need this?

Reply:
somehow when it's `{hasLoadedFromStore}`, it will not print `"true"` but leaving it blank instead, thus failing the test.